tardis/apps/publication_workflow/static/my-publications/my_publications.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * My Publications controller
+ *
+ */
+angular
+  .module('publicationWorkflow')
+  .controller('MyPublicationsController', function ($log, $resource, $timeout) {
+    const vm = this;
+
+    const draftPubsListRes = $resource('/apps/publication-workflow/draft_pubs_list/', {}, {
+      get: { method: 'GET', isArray: true },
+    });
+    const scheduledPubsListRes = $resource('/apps/publication-workflow/scheduled_pubs_list/', {}, {
+      get: { method: 'GET', isArray: true },
+    });
+    const releasedPubsListRes = $resource('/apps/publication-workflow/released_pubs_list/', {}, {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
        const releasedPubsListRes = $resource('/apps/publication-workflow/released_pubs_list/', {}, {
```